CRM/Core/Controller.php

```diff
@@ -875,7 +875,7 @@ public function invalidKeyRedirect() {
         if (!empty($url_parts['query'])) {
           $redirect_url .= '?' . $url_parts['query'];
         }
-        CRM_Core_Session::setStatus(ts('Your browser session has expired and we are unable to complete your form submission. We have returned you to the initial step so you can complete and resubmit the form. If you experience continued difficulties, please contact us for assistance.'));
+        CRM_Core_Session::setStatus(ts('Sorry, your session has expired. Please resubmit your information to complete your submission.'));
```

Comment:
The "Please resubmit" sounds a bit odd to me, oddly specific.  "Please try again"?